teams/archive/wg-binary-size.toml

```diff
@@ -3,24 +3,10 @@ kind = "working-group"
 subteam-of = "compiler"
 
 [people]
-leads = ["thomcc", "m-ou-se"]
-members = [
-    "davidtwco",
-    "h1467792822",
-    "Kobzol",
-    "kupiakos",
-    "m-ou-se",
-    "nbdd0121",
-    "nnethercote",
-    "thomcc",
-    "wain303009",
-    "workingjubilee",
-    "antoyo",
-    "tgross35",
-]
+leads = []
+members = []
 
-alumni = [
-]
+alumni = []
```

Comment:
When archiving teams, we actually keep the alumni :sweat_smile: We should probably write some guide for this.